cmd/action/ci/ci.go

```diff
@@ -61,5 +64,49 @@ func setCfgName(cfg *v1alpha1.Environment) {
 	if len(sha) > 8 {
 		sha = sha[:8]
 	}
-	cfg.Name = fmt.Sprintf("ci%s-%s", attempt, sha)
+	// uid is unique for each run
+	uid := generateUID()
+
+	cfg.Name = fmt.Sprintf("ci%s-%s-%s", attempt, sha, uid)
+}
+
+func generateUID() string {
+	const charset = "abcdefghijklmnopqrstuvwxyz0123456789"
+
+	b := make([]byte, 8)
+	for i := range b {
+		b[i] = charset[rand.Intn(len(charset))]
```

Comment:
Question: Is the lint error reported here or at the import?

Reply:
It would be reported here. Tested in the device plugin:
```
 ✗ make check
golangci-lint run ./...
tests/e2e/framework/framework.go:149:53: G404: Use of weak random number generator (math/rand instead of crypto/rand) (gosec)
                f.UniqueName = fmt.Sprintf("%s-%08x", f.BaseName, rand.Int31())
                                                                  ^
tests/e2e/framework/util.go:50:22: G404: Use of weak random number generator (math/rand instead of crypto/rand) (gosec)
        return strconv.Itoa(rand.Intn(10000))
                            ^
make: *** [Makefile:81: lint] Error 1
```

Reply:
done

Reply:
I mean, do we want to add the nolint here?

Reply:
I did a run without the comment on both 
```bash
❯ golangci-lint run ./... |grep weak
```
and `golangci-lint`  didn't complained

Reply:
I have edited it to only have the lint comment on the code and not the import.

Reply:
This project doesn't seem to have a `.golangci` file. Could that be why we're not seeing the issue?